src/Controller/ItemController.php

```diff
@@ -204,4 +206,19 @@ public function autocomplete(string $search, ItemRepository $itemRepository, Pac
 
         return new JsonResponse($data);
     }
+
+    #[Route(path: '/items/{id}/generate-dublincore-xml', name: 'app_item_generate_dublin_core_xml', methods: ['GET', 'POST'])]
+    public function generateDublinCoreXML(Request $request, Item $item, DublinCoreXMLGenerator $dcGenerator): Response
+    {
+        $generatedXML = $dcGenerator->generateDublinCoreXML($item, $request->getSchemeAndHttpHost());
+
+        $response = new Response($generatedXML['content']);
+        $disposition = HeaderUtils::makeDisposition(
+            HeaderUtils::DISPOSITION_ATTACHMENT,
+            $generatedXML['filename']
```

Comment:
Please consider opening the file in a new tab instead of downlaoding it (especially if you decide to make this option public)

Reply:
That makes sense, done.